geniza/corpus/management/commands/export_annotations.py

```diff
@@ -0,0 +1,143 @@
+            # create and save one AnnotationList per canvas
+            # with all associated annotations, as a backup
+            # that could be imported into any w3c annotation server
+
+            for canvas, annotations in annos_by_canvas.items():
+                annolist_name = self.annotation_list_name(canvas)
+                annolist_out_path = os.path.join(output_dir, "%s.json" % annolist_name)
+                print(annolist_out_path)
+                with open(annolist_out_path, "w") as outfile:
+                    json.dump(
+                        annotations_to_list(annotations, uri="test"), outfile, indent=2
+                    )
+
+            # for convenience and more readable versioning, also generate
+            # text and html transcription files
+            for edition in document.digital_editions():
+                print(edition)
+                # filename should be based on pgpid and source
+                # use source id for now; slug todo
+                # should we explicitly label as transcription?
+                base_filename = "%s_source_%s" % (document.id, edition.source.id)
+                for output_format in ["txt", "html"]:
+                    # put in the appropriate transription dir by format,
+                    # use format as file extension
+                    outfile_path = os.path.join(
+                        transcription_output_dir[output_format],
+                        "%s.%s" % (base_filename, output_format),
+                    )
+                    print(outfile_path)
+                    # TODO: use a minimual django template here;
+                    # should include full citation for the source + footnote.
+
+                    # FIXME: where should primary language lang code be set,
+                    # for this export and for transcription on site?
+
+                    with open(outfile_path, "w") as outfile:
+                        if output_format == "html":
+                            outfile.write(edition.content_html_str)
+                        else:
+                            # TODO: template for this also?
+                            # need to include source citation
+                            # and line numbers for ol list items
+                            outfile.write(edition.content_text)
+
+    def annotation_list_name(self, canvas_uri):
+        # per annotation spec, annotation list name must uniquely distinguish
+        # it from  all other lists, and is typically the same name as the canvas.
+
+        # canvas uris vary depending on source:
+        # https://cudl.lib.cam.ac.uk/iiif/MS-TS-NS-00321-00008/canvas/1
+        # https://figgy.princeton.edu/concern/scanned_resources/f9eb5730-035c-420a-bf42-13190f97c10d/manifest/canvas/cfd65bb6-7ff5-47e8-9e92-29dd0e05baf2
+        # https://princetongenizalab.github.io/iiif-bodleian-a/manifests/canvas/1.json
+        # https://digi.ub.uni-heidelberg.de/diglit/iiif/codheidorient78/canvas/0001
```

Comment:
So this produces something like this? `heidelberg_codheidorient78_canvas_0001`

Maybe examples of output in the comment here would be helpful.